vita3k/config/src/config.cpp

```diff
@@ -133,6 +133,7 @@ static ExitCode parse(Config &cfg, const fs::path &load_path, const std::string
     get_yaml_value(config_node, "color-surface-debug", &cfg.color_surface_debug, false);
     get_yaml_value(config_node, "hardware-flip", &cfg.hardware_flip, false);
     get_yaml_value(config_node, "performance-overlay", &cfg.performance_overlay, false);
+    get_yaml_value(config_node, "backend", &cfg.backend, std::string("OpenGL"));
```

Comment:
Love the string config. People can edit the backend easier in config

Reply:
Yep. Zang wanted it to be a number. In new commit it is kinda both...